pkgs/applications/blockchains/lightning-terminal/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
```

Comment:
The CI log says:
Please go to the Nixpkgs root directory, run `nix-shell`, then:
nixfmt 'pkgs/applications/blockchains/lightning-terminal/default.nix'

The formatter was "recently" updated. You probably found some older formatters.